### DISCUSSION TITLE
This is better python

### DISCUSSION THREAD

**Discussion by BartuV** [google/rune#15] [General]
I want to try it so badly because it's meet my expectation from python with curly braces

**Comment by waywardgeek** [comment]
Don't get too excited.  Python is fun, and always has been.  Rune is still buggy and the compiler error messages can be confusing.  Also, Python is more powerful than Rune in many ways, e.g. Python supports lists containing objects of different types.  Rune compromises in many places to be fast, while Python never really compromises anything, at least not for speed.
